src/main/scala/kamon/newrelic/metrics/NewRelicMetricsReporter.scala

```diff
@@ -29,8 +29,11 @@ class NewRelicMetricsReporter(sender: MetricBatchSender = NewRelicMetricsReporte
     val counters = snapshot.counters.flatMap { counter =>
       new CounterConverter().convert(periodStartTime, periodEndTime, counter)
     }
+    val gauges = snapshot.gauges.flatMap { gauge =>
+      new GaugeConverter().convert(periodEndTime, gauge)
```

Comment:
consider making this an object method, rather than creating lots of useless instances.